system/t06_publish/PublishUpdate6Test_gold

```diff
@@ -1 +1,8 @@
-ERROR: unable to update: not a local repository publish
```

Comment:
this test was supposed to fail, no ?

Reply:
This test case was dropped, because `update` now also works for published snapshot repositories. Before the change, it could only be applied to published local repositories.